packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -217,10 +217,9 @@ const isSpecsListOpenPreferences = computed(() => {
 
 const hideCommandLog = runnerUiStore.hideCommandLog
 
-// watch active spec, and re-run if it changes!
-startSpecWatcher()
-
 onMounted(() => {
+  // watch active spec, and re-run if it changes!
+  startSpecWatcher()
```

Comment:
I wonder why we needed to wait until the element was rendered to call this?

Reply:
this is left over from attempts in trying to correct this behavior in open mode. changing back for now. but my original thought was there wasn't any. need to watch if a spec was changes from a user in the app if the content wasn't mounted & rendered yet.